### PR TITLE
add erbium page

### PR DESCRIPTION
**Pull request by trishakholiya**
## Description
Add erbium page

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] TODO 1

## Questions
- [x] Question1

## Status
- [x] Ready to go

### CODE REVIEW

File: elements/erbium.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: erbium
```
Comment (vincent-nguyen-sys) [inline elements/erbium.md:3]: Looks great. We should capitalize Erbium here to keep is consistent with the fact.

Comment (trishakholiya) [reply elements/erbium.md]: @vincent-nguyen-sys it should be changed now! Can you review?

**Review by JXDolan — COMMENTED** [review]
Everything looks good

**Review by JXDolan — APPROVED** [review]
good to go

**Pull request merged**